webapp/components/CustomIconComponent.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Button from "@material-ui/core/IconButton";
+import RemoveRedEyeIcon from "@material-ui/icons/RemoveRedEye";
+
+const IconComponent = ({ action, color, tooltip }) => (
+  <Button
+    style={{ color }}
+    className="list-icon"
+    title={tooltip}
+    onClick={action}
+  >
+    <RemoveRedEyeIcon />
```

Comment:
What about parametrizing the icon? Otherwise I'd choose a more specific name for this component